### DISCUSSION TITLE
Fulfillment Service

### DISCUSSION THREAD

**Discussion by foresttoney** [saleor/saleor#11053] [Q&A]
It doesn't look like Saleor has an equivalent abstraction to [Shopify's Fulfillment Service](https://shopify.dev/apps/fulfillment/fulfillment-service-apps). I haven't done a deep dive, but at a cursory look it looks like most of the functionality is available via the API. How does the community generally approach this if they were considering a move from Shopify?

**Comment by karolkielecki** [comment]
Hi @foresttoney, You can create an integration (app) with other services. You can find the info on how to do that here: https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts. Could you please tell us your use case? 

Cheers,
